### ISSUE TITLE
Meta site implementation

### ISSUE THREAD

**Issue by AntonioMateoGomez** [arrow-kt/arrow-meta#39]
We need to implement a separate site for the Arrow Meta library. Once done this, we will implement the same way with the other Arrow libraries.

**Comment by rachelcarmena** [comment]
@AntonioMateoGomez , we are already publishing snapshots: https://oss.jfrog.org/artifactory/oss-snapshot-local/io/arrow-kt/

```
arrow-meta-compiler-plugin/                   30-Oct-2019 18:11    -
arrow-meta-docs/                              30-Oct-2019 18:11    -
arrow-meta-idea-plugin/                       30-Oct-2019 18:11    -
arrow-meta-testing-plugin/                    30-Oct-2019 18:11    -
```

**Comment by calvellido** [comment]
Closed in #45
